src/deluge/dsp/compressor/rms_feedback.cpp

```diff
@@ -79,8 +79,8 @@ void RMSFeedbackCompressor::render(StereoSample* buffer, uint16_t numSamples, q3
 		currentVolumeL += amplitudeIncrementL;
 		currentVolumeR += amplitudeIncrementR;
 		// Apply post-fx and post-reverb-send volume
-		thisSample->l = multiply_32x32_rshift32(thisSample->l, currentVolumeL) << 2;
-		thisSample->r = multiply_32x32_rshift32(thisSample->r, currentVolumeR) << 2;
+		thisSample->l = multiply_32x32_rshift32(thisSample->l, currentVolumeL) << 4;
```

Comment:
We can potentially apply 3 compressors to a sound (kit row, kit, song) which can each multiply by 8 for a total possible gain of 2^9 which is a bit dicy

I'm not sure the exact level that samples/waves render at but I think it's 2^20 or something which would definitely clip 

On the other hand it can be fixed by lowering volumes so we could chalk that up to user error